front/app/components/admin/ContentBuilder/Widgets/AccordionMultiloc/index.tsx

```diff
@@ -30,9 +31,16 @@ interface AccordionProps {
 const Accordion = ({ text, title, openByDefault = false }: AccordionProps) => {
   const theme = useTheme();
   const localize = useLocalize();
+  const isSmallerThanTablet = useBreakpoint('tablet');
```

Comment:
All homepage components now have padding of `20px` on smaller than tablet viewport.